packages/create/src/generators/app-lit-element-ts/templates/_MyApp.ts

```diff
@@ -49,7 +49,7 @@ export class <%= className %> extends LitElement {
     }
   `;
 
-  render() {
+  render(): TemplateResult {
```

Comment:
we should also add this in the component starter, too

Reply:
Could you point me to this file so that I can add the relevant code?